src/generators/invokechaincode.sh.ts

```diff
@@ -63,13 +72,14 @@ export class InvokeChaincodeShGenerator {
                 l(`At least one peer returned an error!`);
                 l(`This may happen when a transaction queries private data that's not accessible to all peers`);
                 ex.responses.map(response => {
-                    l(`Response from ${response.peer.name}`);
-                    if (response.isProposalResponse) {
-                        l(JSON.stringify(response));
-                    } else {
-                        // Good response
-                        l(response.response.payload.toString('utf8'));
-                    }
+                    console.log(response);
+                    //l(`Response from ${response.peer.name}`);
+                    //if (response.isProposalResponse) {
+                    //    l(JSON.stringify(response));
+                    //} else {
+                    //    // Good response
+                    //    l(response.response.payload.toString('utf8'));
+                    //}
```

Comment:
Why this change?

Reply:
I just forgot to clean up. Note that the "peer" field is not always available in the response object.